README.md

```diff
@@ -1,342 +1,14 @@
-
-```sbt
-sourceDirectory in Paradox := sourceDirectory.value / "doc"
-```
-
-You may also change the [base-url](https://gohugo.io/overview/configuration/) that gets passed to the `hugo` command by adjusting the following setting:
-
-```sbt
-baseURL in Hugo := "https://yourdomain.com"
-```
-
-## ScalaDoc APIs
-To include ScalaDoc with your site, add the following line to your `build.sbt`:
-
-```sbt
-enablePlugins(SiteScaladocPlugin)
-```
-
-This will default to putting the ScalaDoc under the `latest/api` directory on the website. You can change this with the `siteSubdirName` key in the `SiteScaladoc` scope:
-
-```sbt
-// Puts ScalaDoc output in `target/site/api/wip`
-siteSubdirName in SiteScaladoc := "api/wip"
-```
-
-## Publishing to GitHub Pages
-
-See the [sbt-ghpages] plugin for information about publishing to [GitHub Pages]. We expect other publishing mechanisms to be supported in the future.
-
-## Previewing the Site
-To preview your generated site, you can run `previewSite` which launches a static web server, or `previewAuto` which launches a dynamic server updating its content at each modification in your source files. Both launch the server on port 4000 and attempts to connect your browser to [http://localhost:4000/](http://localhost:4000/). To change the server port, use the key `previewFixedPort`:
-
-```sbt
-previewFixedPort := Some(9999)
-```
-
-To disable browser auto-open, use the key `previewLaunchBrowser`:
-
-```sbt
-previewLaunchBrowser := false
-```
-
-## Packaging and Publishing
-To create a zip package of the site run `package-site`.
-
-To also include this zip file as an artifact when running `publish`, add the following to your `build.sbt`:
-
-```sbt
-publishSite
-```
-
-See the [sbt-ghpages] plugin documentation for simplified publishing to [GitHub Pages].
-
-## Advanced Usage
-
-If you need to run a generator on more than one source directory, bypassing the `AutoPlugin` system and defining one or more sbt `Configuration`s is necessary. For example, suppose you have two Pamflet source directories and want them each generated as a subdirectory under `target/site`. A `build.sbt` might look something like this:
-
-```sbt
-// Define two `Configuration` instances.
-val Site1 = config("site1")
-
-val Site2 = config("site2")
-
-// Apply the default pamflet settings to the `Site1` config
-PamfletPlugin.pamfletSettings(Site1)
-
-// Customize the source directory
-sourceDirectory in Site1 := sourceDirectory.value / "pamflet-site-1"
-
-// Customize the output subdirectory
-siteSubdirName in Site1 := "chapter1"
-
-// Same as above, but for config `Site2`
-PamfletPlugin.pamfletSettings(Site2)
-
-sourceDirectory in Site2 := sourceDirectory.value / "pamflet-site-2"
-
-siteSubdirName in Site2 := "chapter2"
-```
+## Documentation
 
-Each of the other generators follow a similar pattern (e.g. `JekyllPlugin.jekyllSettings(config("foo"))`).
+The documentation is available at <https://sbt.github.io/sbt-site/>
```

Comment:
Not sure if it would be possible to somehow (e.g. from Travis) publish it under http://www.scala-sbt.org/sbt-site/ similar to http://www.scala-sbt.org/sbt-native-packager/ ?

/cc @dwijnand @eed3si9n